doc/whats_new/v0.22.rst

```diff
@@ -107,6 +107,13 @@ Changelog
   preserve the class balance of the original training set. :pr:`14194`
   by :user:`Johann Faouzi <johannfaouzi>`.
 
+- |Fix| Enable to run :func:`utils.estimator_checks.check_estimator` on both
```

Comment:
I would say "run it by default" because you could already run it by giving it an instance.